baybe/kernels/__init__.py

```diff
@@ -0,0 +1,6 @@
+"""Kernels that can be used for GP surrogate models."""
+
+from baybe.kernels.base import Kernel
+from baybe.kernels.kernels import MaternKernel
```

Comment:
I am not happy with the naming here, do you have other ideas of how the stuff could be called to avoid `kernels.kernels`?

Reply:
How about "basic"?

Reply:
+1

Reply:
Will do this at the very end when rebasing and stuff like that and will keep that comment as merge-blocker. In particular, feel free to approve the PR if this is the only open conversation remaining :)